GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2467,11 +2467,6 @@ public static void OpenContainingFolder(FileStatusList diffFiles, GitModule modu
             foreach (var item in diffFiles.SelectedItems)
             {
                 string filePath = PathUtil.Combine(module.WorkingDir, item.Item.Name.ToNativePath());
-                if (string.IsNullOrWhiteSpace(filePath))
-                {
-                    continue;
-                }
-
```

Comment:
All other changes make sense, but this change is surprising in the given context. Could you please elaborate on this?

Reply:
null annotation is coming soon...
null is an acceptable input with the change so the caller do not need to check